undertow/WFLY-11764_UndertowOptions_PRESERVE_PATH_ON_FORWARD.doc

```diff
@@ -0,0 +1,82 @@
+= WFLY-11764 Make return value of HttpServletRequest.getServletPath and HttpServletRequest.getRequestURI configurable (UndertowOptions.PRESERVE_PATH_ON_FORWARD)
+:author:            Flavia Rainone
+:email:             frainone@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          getServletPath,getRequestURI,preserve path
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This feature is to support a backwards compatible behavior of HttpServletRequest.getServletPath and HttpServletRequest.getRequestURI
+ in Undertow. In an Struts2 application deployed to EAP6, that method returns the action name, but in EAP7, it returns the jsp name.
+A new UndertowOption was created to support that behavior (UndertowOptions.PRESERVE_PATH_ON_FORWARD) and now we need to make that
+behavior configurable in EAP7.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-11764[WFLY-11764]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1341[EAP7-1341]
+* https://issues.jboss.org/browse/JBEAP-17141[JBEAP-17141]
+* https://issues.jboss.org/browse/UNDERTOW-1159[UNDERTOW-1159]
+* https://issues.jboss.org/browse/UNDERTOW-1499[UNDERTOW-1499]
+* https://issues.jboss.org/browse/UNDERTOW-1504[UNDERTOW-1504]
+* https://issues.jboss.org/browse/JBEAP-16448[JBEAP-16448]
+* https://issues.jboss.org/browse/JBEAP-16449[JBEAP-16449]
+
+
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+* mailto:bspyrkos@redhat.com[Bartosz Spyrko-Smietanko]
+
+=== QE Contacts
+
+* mailto:jstourac@redhat.com[Jan Stourac]
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+[ ] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+* WildFly, Undertow component
+* Undertow
+
+=== Other Interested Projects
+
+
+== Requirements
+
+This will be implemented as an additional attribute under the `subsystem=undertow` management model address.
```

Comment:
actually under the `/subsystem=undertow/servlet-container=<name>`?

Reply:
@jstourac  Good point, I'll update